source/features/submit-review-as-single-comment.tsx

```diff
@@ -56,7 +56,7 @@ async function handleSubmitSingle(event: delegate.Event): Promise<void> {
 	const commentText = select('textarea[name="pull_request_review_comment[body]"]', commentContainer)!.value;
 	if (!commentText) {
 		alert('Error: Comment not found and not submitted. More info in the console.');
-		features.log.error(__filebasename, 'Comment not found');
+		features.log.error(features.getFeatureID(import.meta.url), 'Comment not found');
```

Comment:
`log.error` should also be updated to accept the raw url

```suggestion
		features.log.error(import.meta.url, 'Comment not found');
```